src/libraries/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -79,13 +79,23 @@ add_library(clrcompression
     ${CMAKE_REPO_ROOT}/artifacts/obj/NativeVersion.rc
 )
 
+add_library(clrcompression-static
+    STATIC
+    ${NATIVECOMPRESSION_SOURCES}
+)
+
+SET_TARGET_PROPERTIES(clrcompression-static PROPERTIES PREFIX "")
+SET_TARGET_PROPERTIES(clrcompression-static PROPERTIES OUTPUT_NAME libclrcompression)
```

Comment:
@NextTurn I have a couple of questions:
* Why do we add the lib prefix on Windows? This naming convention is Unix only.
* If we really want to do that for some reason, why don't we use the PREFIX property to do that instead of setting the PREFIX to an empty string and modifying the OUTPUT_NAME?

Reply:
> Why do we add the lib prefix on Windows? This naming convention is Unix only.

There is already a *clrcompression.lib* on Windows serving as an import library. The `lib` prefix is taken from a similar library *libnethost.lib* as per discussion in dotnet/core-setup#8659.

> If we really want to do that for some reason, why don't we use the PREFIX property to do that instead of setting the PREFIX to an empty string and modifying the OUTPUT_NAME?

Setting the PREFIX to `lib` seems to work as well... I'll take another look later.

Reply:
How about we set PREFIX to `lib` and then OUTPUT_NAME to `clrcompression`?

Reply:
Hmm, I have not realized before that we need to set the output name anyways, as it would be clrcompression-static by default. So I guess it is not worth the hassle to spend time changing it to set the PREFIX to lib. Please feel free to keep it as is.